pkgs/development/interpreters/perl/default.nix

```diff
@@ -179,6 +193,7 @@ let
 in rec {
   # the latest Maint version
   perl528 = common {
+    self = perl528;
```

Comment:
```perldevel``` few lines below should have ```self = perldevel``` added too

perhaps there are more problems, this one was easy to spot

Reply:
ofborg failed here too :)